public/locales/fi/signup.json

```diff
@@ -107,6 +108,7 @@
     "placeholderFirstName": "Syötä ilmoittautujan etunimi",
     "placeholderLastName": "Syötä ilmoittautujan sukunimi",
     "placeholderPhoneNumber": "+358 44 1234 567",
+    "placeholderPriceGroup": "Valitse hintaryhmä",
```

Comment:
Here possibly "asiakasryhmä" should be used

Reply:
Changed "Hintaryhmä" to "Asiakasryhmä"